homeassistant/components/ccl/entity.py

```diff
@@ -0,0 +1,60 @@
+"""The mapping of a CCL Entity."""
+
+from __future__ import annotations
+
+from aioccl import CCLDevice, CCLSensor
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import CCLCoordinator
+
+
+class CCLEntity(CoordinatorEntity, Entity):
```

Comment:
```suggestion
class CCLEntity(CoordinatorEntity[..]):
```
Add the coordinator type to the typeslot